docs/examples/cloudauditlogssource/README.md

```diff
@@ -148,13 +148,6 @@ Data,
   }
 ```
 
-## What's Next
```

Comment:
hmm I think we should keep this... I was referring some sentences in the description of CloudPubSubSource